client/src/app/videos/+video-watch/comment/video-comments.component.scss

```diff
@@ -19,6 +19,11 @@
   font-size: 13px;
 }
 
+.comment-help {
+  display: inline-block;
+  margin-left: -50px;
```

Comment:
I don't really like setting negative margins. Could you try to put the help icon in the `title-page` block to avoid that?